test/functional/helpers.lua

```diff
@@ -417,8 +418,12 @@ local function curtab(method, ...)
   return tabpage(method, 0, ...)
 end
 
-local function expect(contents)
-  return eq(dedent(contents), curbuf_contents())
+local function expect(contents, any)
```

Comment:
If this is really needed, let's make it a new function `expect_any()`. The reason we didn't do that for `screen:expect()` is because that function is difficult to refactor.